samples/wifi/provisioning/implementation_details.rst

```diff
@@ -0,0 +1,268 @@
+| Point           |             |             |             |             |
++-----------------+-------------+-------------+-------------+-------------+
+| Operation       | M           | Read, Write |             | Encryption  |
+| Control         |             |             |             | Required    |
+| Point           |             |             |             |             |
+| - Client        |             |             |             |             |
+| Characteristic  |             |             |             |             |
+| Configuration   |             |             |             |             |
+| descriptor      |             |             |             |             |
++-----------------+-------------+-------------+-------------+-------------+
+| Data Out        | M           | Notify      |             | Encryption  |
+|                 |             |             |             | Required    |
++-----------------+-------------+-------------+-------------+-------------+
+| Data Out        | M           | Read, Write |             | Encryption  |
+| - Client        |             |             |             | Required    |
+| Characteristic  |             |             |             |             |
+| Configuration   |             |             |             |             |
+| descriptor      |             |             |             |             |
++-----------------+-------------+-------------+-------------+-------------+
+
+Information
+-----------
+
+This characteristic is used to get Info message.
+when read, the Information characteristic gives an encoded Protocol Buffers message in type of Info defined in version.proto.
+It is used by client to determine which version of Wi-Fi provisioning service is running on this server.
+
+Operation Control Point
+-----------------------
+
+This characteristic is used to send Request message and get Response message.
+The client shall send an encoded message in type of Request defined in request.proto, and the server shall send an encoded message in type of Response defined in response.proto.
+
+Data Out
+--------
+
+This characteristic is used to send notifications to the client in format of Result message.
+
+Advertisement
+-------------
+
+After power up, the device will first check if it is provisioned or not.
+If so, it will load saved credentials and try to connect the AP specified in the credential.
+After 30 seconds (timeout will occur if the attempt fails), it will start BLE advertising.
+If it is not provisioned, it will advertise immediately.
+
+The advertising data contains several fields to facilitate possible provisioning attempt.
+It will contain the UUID of Wi-Fi provisioning service, and a 4-byte service data.
+
++----------+----------+-----------+----------+
+|  Byte 1  |  Byte 2  |   Byte 3  |  Byte 4  |
++==========+==========+===========+==========+
+| Version  |         Flag         |   RSSI   |
```

Comment:
Mark bytes 2 and 3 both as Flag in separate cells, instead of one. 

The text aligns to the left and the render makes it look like this is a mistake.